std/root.lnn

```diff
@@ -0,0 +1,19 @@
+// note: most-recently-defined wins as tie-breaker
```

Comment:
don't we have multiple dispatch or is there a case when a function has the same name and arguments?

Reply:
Even when they have different argument types it could come into play. If one function has an interface type and another a *different* interface type or an actual type and they both match, choosing the most-recently-defined makes it possible for users to override behavior from imported modules while anything else becomes effectively "unpredictable" (if it's based on which is the "best match" the developer needs to know those rules, and there still needs to be a tiebreaker rule there, too).

Reply:
this is behavior that is unchanged from the current front-end. I just left this comment here to remind myself :)